lib/core/docs/storiesOf.md

```diff
@@ -31,6 +31,8 @@ storiesOf('Button', module)
 
 The string argument to `storiesOf` is the component title. If you pass a string like `'Widgets|Button/Button'` it can also be used to position your component's story within Storybook's story hierarchy.
 
+The second argument to `storiesOf` is the module object, the one you might be familiar with from ES5's `module.exports`. Storybook needs a reference to the file/module where your story lives to enable hot-module-replacement. If you do not supply it, you'd need to refresh your browser for every change you make.
+
```

Comment:
@neilyio we have a bit of room for improvement here. We could probably go with:

" The second argument of `storiesOf` is a reference to each story available. Storybook needs it to enable hot-module-replacement. If it's not included you'll need to refresh your browser with each change you make."

Let me know if you're ok with it.